PerfCounterReporter/CounterDefinitions/system.counters

```diff
@@ -80,4 +80,7 @@
 #bottleneck. This counter is a sum of �Network Interface\\Bytes Received/sec� and 
 #�Network Interface\\Bytes Sent/sec�.  In some situations, you may wish to determine 
 #both inbound and outbound network traffic separately
-\Network Interface(*)\Bytes Total/sec
\ No newline at end of file
+\Network Interface(*)\Bytes Total/sec
+
+\LogicalDisk(*)\Free Megabytes
+\LogicalDisk(*)\% Free Space
```

Comment:
In windows any application cares only about space in Logical Volume because it can write only to it. Thats why there is no such concept as Physical Disk Free Space. Having those two metrics is enough to calculate occupied space on a volume. As to total Free/Occupied space on a disk we can get it by summing up all volumes per disk.